.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - UX Audit
+# Label to use when marking an issue as stale
+staleLabel: wontfix
```

Comment:
Any other label we want to use?

Reply:
wontfix isn't the right sentiment, that implies that we acknowledge it's a problem. How about 'inactive'?

Reply:
```suggestion
staleLabel: inactive
```